### PR TITLE
Testing.rst: Add directions about getting started

### PR DESCRIPTION
**Pull request by jschwarzwalder**
Testing.rst: Add directions about getting started

### CODE REVIEW

**Comment by sils1297** [comment]
Hi,

it looks like your PR is out of date and needs a rebase. [This page](http://coala.readthedocs.org/en/latest/Users/Tutorials/Git_Help.html) may help you to get started on this. (*Do not press the update branch button. It's evil.*)

Add the original coala repository as a remote:

```
git remote add upstream https://github.com/coala-analyzer/coala.git
```

Then go to your branch and rebase it onto our master:

```
git checkout <yourbranch>
git fetch upstream
git rebase --interactive upstream/master
```

Now push it up. You will need to do a force push because you change history here (do not do this on master branches :)):

```
git push --force
```

Ping us on [our gitter channel](https://gitter.im/coala-analyzer/coala) if you need help!

**Comment by sils1297** [comment]
or maybe @underyx can help you again, since he already knows the graphics he had drawn for you

**Comment by Makman2** [comment]
please remove the "test report" commit, it's a plain png and not related to the docs^^

File: docs/Getting_Involved/Testing.rst [hunk 1]
```diff
@@ -9,8 +9,15 @@ Executing our Tests
 *coala* has a big test suite. It is meant to work on every platform on
 every PC. If you just execute our tests you are doing us a favor.
 
+If you have not already, you can start by following
+the `coala Tutorial <Users/Tutorials/Tutorial.html>`_.
+
+Before you run tests, make sure you
+`install coala
+<Users/Install.html/>`_.
```
Comment (Makman2) [inline docs/Getting_Involved/Testing.rst]: please wrap this into the previous line :+1:, and if possible even in the line before the previous one

File: docs/Getting_Involved/Testing.rst [hunk 2]
```diff
@@ -9,8 +9,15 @@ Executing our Tests
 *coala* has a big test suite. It is meant to work on every platform on
 every PC. If you just execute our tests you are doing us a favor.
 
+If you have not already, you can start by following
+the `coala Tutorial <Users/Tutorials/Tutorial.html>`_.
```
Comment (Makman2) [inline docs/Getting_Involved/Testing.rst]: you can put "the" into the previous line :+1:

File: docs/Getting_Involved/Testing.rst [hunk 3]
```diff
@@ -9,8 +9,15 @@ Executing our Tests
...
+
+Before you run tests, make sure you
+`install coala
+<Users/Install.html/>`_.
+
 To run tests, you first need to install some dependencies. This can be
-done by executing:
+done by executing in your coala directory:
 
 ::
```
Comment (Makman2) [inline docs/Getting_Involved/Testing.rst]: Does this commit fix an issue?

File: docs/Getting_Involved/Testing.rst (hunk 1 above)
Comment (Makman2) [inline docs/Getting_Involved/Testing.rst]: btw you don't need to :O tests work without coala installed (at least they should)

File: docs/Getting_Involved/Testing.rst (hunk 2 above)
Comment (Makman2) [inline docs/Getting_Involved/Testing.rst]: \+ link not working^^

File: docs/Getting_Involved/Testing.rst (hunk 3 above)
Comment (Makman2) [inline docs/Getting_Involved/Testing.rst]: \+ for the commit message: we use normally a tag inside the shortlog (normally the filename modified). Also your shortlog should include an imperative tense verb describing the change, e.g.

`Testing.rst: Add directions about getting started`

**Comment by Adrianzatreanu** [comment]
@jschwarzwalder hey, you changed the title of the Pull Request, not of the commit. to change the shortlog of the commit you should use ``git commit --amend``. Also, seems you need a rebase :)

File: docs/Getting_Involved/Testing.rst [hunk 4]
```diff
@@ -9,8 +9,14 @@ Executing our Tests
 *coala* has a big test suite. It is meant to work on every platform on
 every PC. If you just execute our tests you are doing us a favor.
 
+If you have not already, you can start by following the
+`coala Tutorial <Users/Tutorials/Tutorial.html>`_.
```
Comment (sils1297) [inline docs/Getting_Involved/Testing.rst]: how is the user tutorial relevant for developers running tests?

File: docs/Getting_Involved/Testing.rst [hunk 5]
```diff
@@ -9,8 +9,14 @@ Executing our Tests
...
+
+Before you run tests, make sure you `install coala
+<Users/Install.html/>`_.
```
Comment (sils1297) [inline docs/Getting_Involved/Testing.rst]: hm, tests should work without installation

**Comment by sils1297** [comment]
Hey, based on your work and our discussion I made https://github.com/coala-analyzer/coala/pull/2350 , which is an alternative to your PR. Do you think it's an acceptable comprimise between brevity and newcomer friendliness?

**Comment by sils1297** [comment]
marking WIP as there's some discussion still to happen here

**Comment by Makman2** [comment]
ping

**Comment by Adrianzatreanu** [comment]
hey @jschwarzwalder i know you are there since you've been working on other PRs. i hope you havent forgotten this :D

**Comment by jschwarzwalder** [comment]
@Adrianzatreanu @Makman2  I've not forgotten about it. I just am not sure what steps I can take. Both pull requests have stagnated. On the other one Lasse wanted more feedback for what action to take.

Many of the pull requests I'd been working on submitting was focusing on prerequisites as defined in #2350

Both are still open. I don't know if I should make the changes that Lasse had made in this PR or close this one. I was waiting for more comments on #2350 before closing or adding to this PR.

**Comment by sils1297** [comment]
Hey @Adrianzatreanu can you look into this and see what's needed so this can continue? AFAIK @jschwarzwalder is waiting for input from us here.

**Comment by Adrianzatreanu** [comment]
We moved the documentation here https://github.com/coala-analyzer/documentation and we'll be working there until it's under some reworking and removing redundant stuff, so we will stall this PR for now, until it is done :) sorry @jschwarzwalder

**Comment by Makman2** [comment]
@Adrianzatreanu done reworking? :)

**Comment by Adrianzatreanu** [comment]
yes, imo this should be moved to documentation as well, and adapted to what we got now :)

**Comment by Makman2** [comment]
Migrated here: https://github.com/coala-analyzer/documentation/pull/102

**Pull request closed without merging**